README.md

```diff
@@ -1,6 +1,10 @@
 # dice-cards
 
-A fairer way to provide 2d6 dice rolls
+### A fairer way to provide 2d6 dice rolls!
+
+This GitHub project focuses on enhancing the gaming experience of Settlers of Catan, a board game where players build settlements and cities on a board with numbered tiles representing resources. Players roll 2d6 (2 6-sided dice) to determine which tiles produce resources each turn.
```

Comment:
Games like settlers of Catan, not only that game. Other similar games would also benefit from this mechanism.

Reply:
Change the wording to: 

> This GitHub project focuses on enhancing the gaming experience of games that roll 2 6-sided dice (2d6). For example, Settlers of Catan is a board game where...